libnmstate/schema.py

```diff
@@ -257,6 +257,11 @@ class STP:
         MAX_AGE = 'max-age'
         PRIORITY = 'priority'
 
+    class Options:
```

Comment:
It would be nice if you could sort this, i.e. add the Options class before the Port class and also sort the attributes

Reply:
Done!